requests-for-comments/commnunity-2024.md

```diff
@@ -0,0 +1,47 @@
+
+1. There are already numerous places to share content about building startups / engineering topics generally. Why should people post it to ours and not another, more established channel?  
+
+2. The attraction of our content comes from our strong POV and curation. Outsourcing curation to the community undermines a strength we already have.
+
+3. If it were successful, it would create a significant moderation challenge that would suck up a bunch of time. Abuse and self-promotion will happen and we'll have to manage that.
+
+4. There's a significant risk our own content would be drowned out by community contributions and submissions to the point.
+
+5. It doesn't create any kind of stickiness for our product.
+
+## An alternative vision – HogBook
+
+I think the best way to create a thriving community on the website is to focus it around the product. This means:
+
+- Making it easy to request new features and vote on what the teams are working on
+- Making it easy to see what teams are working on and their progress
+- Creating a direct connection between all users and our engineers (not just the teams who pay for a dedicated Slack channel)
+- Creating a feedback loop where users request / vote on a feature then get notified when it's shipped
+- Creating a space where users can share feedback in public, rather than it being hidden in Zendesk / Slack channels.
+
+To this end, I think the community homepage should be more like a Facebook / Twitter-style feed experience comprising:
+
+- Content we've published that's shared by the author (rather than content automatically appearing when it's published, the author would share it like they would on a social network).
+- Feature requests from users and proposals use that people can comment and vote on – much like they currently do via the Roadmap + GitHub.
+- Semi-regular dev diaries / updates from individual teams documenting their progress.
+- Dedicated team pages (effectively like Facebook groups but for our teams)
+
+In this scenario, the sidebar would look something like this:
+
+<img width="684" alt="Screenshot 2023-12-17 at 11 03 14" src="https://github.com/PostHog/posthog.com/assets/92976667/b6f3b6fa-b5b3-468b-ad9f-bd3c260f51d3">
+
+- The team pages would include feature requests, roadmaps, dev diaries from the team, and content relevant to that product.
+- The community homepage would then combine all this into a single feed that combines the content from the team pages and content hubs.
+
+### Random notes:
+
+- Things like customer stories and comparisons etc. wouldn't appear here as they're largely marketing material not intended for the community.
+
+- Would be good if there were more reactions than just upvoting – something closer to Slack-style reactions.
+
+- This would exist somewhat separate to the Questions forum. TL;DR: feature requests and feedback would go in the team pages. Support questions / problems remain in questions.
```

Comment:
Commenting my overall thoughts on this RFC on this last line (so it's easier to reply to):

Overall:
1. I think HogBook is a great idea, and we should do it.
2. I don't think Hogbook and our current direction are mutually exclusive – we can do both.

I see Hogbook as community building for PostHog users, and our current direction as community building for non-PostHog users. If we want to focus on community building for only one of these, we should be clear which one it is and why.

From a marketing perspective, focusing on PostHog users will help with word-of-mouth growth, whereas focusing on non-PostHog users will also help growth. However, I suspect focusing on non-PostHog users would help us grow more, but this is just based on intuition from the top of my head